.pre-commit-config.yaml

```diff
@@ -10,6 +10,11 @@ repos:
           - --expand-star-imports
           - --remove-duplicate-keys
           - --remove-unused-variables
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.8.0
+    hooks:
+      - id: isort
+        exclude: tests/testdata|astroid/__init__.py
```

Comment:
That should probably be fully covered by / move to the `setup.cfg`

Reply:
I don't understand ? We need this because there is a circular import and if isort is applied on ``astroid/__init__.py`` it breaks everything. We'll still have it after we move to setup.cfg. This will probably be a lot of work to fix.

Reply:
🤦🏻 I mixed that on up with the config for `flake8`. There you have a setting `extend_exclude` for the `setup.cfg` so you don't need to specify the excludes in the `.pre-commit-config`.